ui/packages/docs/src/components/AoE/character_data.json

```diff
@@ -1073,6 +1073,90 @@
       }
     ]
   },
+  "charlotte": {
+    "cons": [
+      {
+        "ability": "C6 Heal",
+        "shape": "Circle",
+        "center": "PrimaryTarget",
+        "radius": 3.5
+      },
+      {
+        "ability": "C6 Attack",
+        "shape": "Circle",
+        "center": "PrimaryTarget",
+        "radius": 5
+      }
+    ],
+    "normal": [
+      {
+        "ability": "N1",
+        "shape": "Circle",
+        "center": "PrimaryTarget",
+        "radius": 1
+      },
+      {
+        "ability": "N2",
+        "shape": "Circle",
+        "center": "PrimaryTarget",
+        "radius": 1
+      },
+      {
+        "ability": "N3",
+        "shape": "Circle",
+        "center": "PrimaryTarget",
+        "radius": 2
+      }
+    ],
+    "charge": [
+      {
+        "ability": "CA",
+        "shape": "Circle",
+        "center": "PrimaryTarget",
+        "radius": 2.5
+      }
+    ],
+    "skill": [
+      {
+        "ability": "E-Tap",
+        "shape": "Circle",
+        "center": "Player",
+        "fanAngle": 120,
+        "radius": 6
+      },
+      {
+        "ability": "E-Hold",
+        "shape": "Box",
+        "center": "Player",
+        "offsetX": 1.3,
+        "boxX": 4.5,
+        "boxY": 30
+      },
+      {
+        "ability": "E-Mark",
+        "shape": "Single",
+        "center": "GlobalValue"
+      }
+    ],
+    "burst": [
+      {
+        "ability": "Q-Attack",
+        "shape": "Circle",
+        "center": "Player",
+        "offsetY": 4.5,
+        "radius": 6,
+        "notes": "Q/Kamera area is the same as this hitbox."
+      },
+      {
+        "ability": "Q-Heal",
+        "shape": "Circle",
+        "center": "Player",
+        "offsetY": 4.5,
+        "radius": 7,
+        "notes": "Q/Kamera area is the same as this hitbox."
+      }
```

Comment:
replace this with
```
{
    "ability": "Q",
    "shape": "Circle",
    "center": "Player", 
    "offsetY": 4.5,
    "radius": 6,
    "notes": "Heal is 7m instead of 6m."
}
```